source/css/_schemes/Pisces/_layout.styl

```diff
@@ -1,7 +1,15 @@
+get_max_content_width() {
+  if (hexo-config('scheme') == 'Gemini') {
```

Comment:
If for Gemini only, need to add this in Gemini scheme file maybe?

Reply:
Yeah, maybe like the below?